dashy/src/app/dashboard/widget/divider-widget/divider-widget.component.html

```diff
@@ -0,0 +1,5 @@
+<div class="col-md-12">
+  <h2>{{title}}</h2>
+  <h5>&nbsp;&nbsp;&nbsp;{{description}}</h5>
```

Comment:
Use CSS property `margin-left` on the `h5` tag to indent this heading, instead of using non breaking spaces.